.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Question
+about: Use this Template to ask a question regarding JDA (Joining the Discord is recommendet)
+---
+
+<!--
+  We suggest to join the JDA Discord server for faster responses: https://discord.gg/0hMr4ce0tIl3SLv5
+-->
+
+[download]: https://bintray.com/dv8fromtheworld/maven/JDA/_latestVersion
+[guild]: https://discord.gg/0hMr4ce0tIk3pSjp
+[wiki]: https://github.com/DV8FromTheWorld/JDA/wiki
+[FAQ]: https://github.com/DV8FromTheWorld/JDA/wiki/10%29-FAQ
+
+## General Troubleshooting
+
+<!--
+  Hey there! Before you ask your question, please make sure
+  to follow these steps first!
+-->
+
+- [ ] I have checked for similar issues.
+- [ ] I have updated to the [latest JDA version][download]
+- [ ] I have checked the [wiki] and especially the [FAQ] section for similar questions.
+
+<!--
+  This is not the place to learn Java. Please refer to StackOverflow for your
+  general programming questions: https://stackoverflow.com/questions/tagged/java
+-->
+
+## Question
+
+<!--
+Ask your question here and provide as much information as possible.
```

Comment:
The other notes have indentation.

```suggestion
  Ask your question here and provide as much information as possible.
```

Reply:
This is based on the feature request template which also doesn't have a indent here, so I left it away on purpose.